frontend/app/routing.js

```diff
@@ -79,7 +133,7 @@ angular.module('openproject')
       templateUrl: '/templates/work_packages.list.new.html'
     })
     .state('work-packages.list.details', {
-      url: '/{workPackageId:[0-9]+}?query_props',
+      url: '/details/{workPackageId:[0-9]+}?query_props',
```

Comment:
Ah yeah, missed the nested routes for the show page. I'll fix them once I managed to get the drop-down menu working on the show page.